attune/curve/_topas.py

```diff
@@ -198,7 +199,7 @@ def save(self, save_directory=None, plot=True, verbose=True, full=False):
             self.plot(autosave=True, save_path=image_path, title=title)
 
         while len(to_insert):
-            _, curve = to_insert.popitem()
+            curve = to_insert[list(to_insert.keys())[0]]
```

Comment:
Order matters, only for signal and idler here, because the sibling of the freshly computed curve is the old curve that is being replaced (so both the curve and it's sibling have the same interaction string, and thus crazy things happen)